arrow/locales.py

```diff
@@ -188,24 +245,31 @@ def meridian(self, hour, token):
             return self.meridians["am"] if hour < 12 else self.meridians["pm"]
         if token == "A":
             return self.meridians["AM"] if hour < 12 else self.meridians["PM"]
+        return None
 
-    def ordinal_number(self, n):
+    def ordinal_number(self, n: int) -> str:
         """Returns the ordinal format of a given integer
 
         :param n: an integer
         """
         return self._ordinal_number(n)
 
-    def _ordinal_number(self, n):
+    def _ordinal_number(self, n: int) -> str:
         return f"{n}"
 
-    def _name_to_ordinal(self, lst):
+    def _name_to_ordinal(self, lst: Sequence[str]) -> Dict[str, int]:
         return dict(map(lambda i: (i[1].lower(), i[0] + 1), enumerate(lst[1:])))
 
-    def _format_timeframe(self, timeframe, delta):
-        return self.timeframes[timeframe].format(trunc(abs(delta)))
+    def _format_timeframe(self, timeframe: TimeFrames, delta: Union[float, int]) -> str:
+        # TODO: remove cast
+        return cast(str, self.timeframes[timeframe]).format(trunc(abs(delta)))
```

Comment:
Hey @isac322 do you want to remove this?

Reply:
@systemcatch Yeap, I want it because `cast` is some kind of monkey patch. But I think just leave it on this PR and refactor whole Locale part later is better approach.